src/Scene/BoundingVolume.cpp

```diff
@@ -37,3 +37,26 @@ BoundingBox BoundingBox::Surrounding(const BoundingBox& a, const BoundingBox& b)
                     std::fmax(a.MaxBound.Z, b.MaxBound.Z));
     return BoundingBox(min, max);
 }
+
+void BoundingBox::Rotate(BoundingBox& bounding, double sinA, double cosA, size_t axis) {
```

Comment:
Should likely be called BoundRotated, or something along those lines since it doesn't actually rotate the bounding box